lib/collection/src/shard/local_shard.rs

```diff
@@ -563,4 +561,17 @@ impl Drop for LocalShard {
             }
         }
     }
+
+    pub async fn delete_from_disk(&self) -> CollectionResult<()> {
```

Comment:
were you able to somehow test any of this part?

Reply:
I can add a test if you think it makes sense?

Reply:
I'd love a test as a quick sanity check but only if the test is easy to write for you of course.

Reply:
We decided to postpone the test to the next PR